handlers/handlers.go

```diff
@@ -59,69 +59,85 @@ func read(w http.ResponseWriter, req *http.Request, cfg *config.Config, zc Zebed
 		return
 	}
 
-	apiQuery := data.GetSearchAPIQuery(validatedQueryParams, censusTopicCache)
-
-	var homepageResponse zebedeeCli.HomepageContent
-	var searchResp searchCli.Response
-	var respErr error
+	var (
+		counter            = 3
```

Comment:
I see this is for the wait group but I would add some quick comments explaining what this `counter` variable is for and why we are reducing it by 2 in line 71. It would make it a lot easier to understand if we don't have magic numbers.